### ISSUE TITLE
[Feature] Manage multiple disk games

### ISSUE THREAD

**Issue by crispybegs** [zurdi15/romm#40]
how should we handle games with more than one disk?
e.g,. Alone in the Dark has two disks

i tried
```
> games
>> dc
>>> roms
>>>> Alone in the Dark
>>>>> Alone in the Dark - The New Nightmare (USA) (Disc 1).chd
>>>>> Alone in the Dark - The New Nightmare (USA) (Disc 2).chd
```

but nothing showed up after scanning the folder from the front end. So i tried taking them out of the subfolder and put both disks in the roms folder, thus...

```
> games
>> dc
>>> roms
>>>>> Alone in the Dark - The New Nightmare (USA) (Disc 1).chd
>>>>> Alone in the Dark - The New Nightmare (USA) (Disc 2).chd
```

...but now I have two entries for one game in the front end. 
What's the correct way to handle these?

<img width="1268" alt="Screenshot 2023-03-28 at 16 02 41" src="https://user-images.githubusercontent.com/129169122/228282795-33b80764-386d-4590-a82d-6bda75117c5b.png">

**Comment by zurdi15** [comment]
Hi, for now RomM doesn't support to join multiple disks of a single game in a single entry. It will be added as a feature.

**Comment by zurdi15** [comment]
Just released [v1.6.3](https://github.com/zurdi15/romm/releases/tag/v1.6.3) with multifile games!
